master/packaging/fetch-cloud-forest.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Install native version to ensure it's on PATH
+GO111MODULE=off go install github.com/ryanbressler/CloudForest/growforest
+
+# Ensure the Linux binary is installed regardless of OS
+GO111MODULE=off GOOS=linux go install github.com/ryanbressler/CloudForest/growforest
```

Comment:
I said this elsewhere, but my concern with this is with variations in GOARCH not being accounted for. Also we talked offline but I dont think `GO111MODULE=off` is the way to go for this.

Reply:
Switched to tools.go